src/main/java/com/google/graphgeckos/dashboard/storage/BuildInfo.java

```diff
@@ -0,0 +1,68 @@
+package com.google.graphgeckos.dashboard.storage;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Represents the response json with the aggregated buildbot data.
+ */
+public class BuildInfo {
+
+  // Git commit hash
+  private final String commitHash;
+  // time of the Git commit push
+  private final Date timestamp;
+  // link to to the repository
+  private final String repository;
+  // name of the branch
+  private final String branch;
+
+  // builders
+  private final List<Builder> builders;
+
+  BuildInfo(final String commitHash, final Date timestamp, final String repository,
+                   final String branch, final List<Builder> builders) {
```

Comment:
nit: align the first `final` on this line with the first `final` on the previous.